bridge/client/app/_views/ktb-environment-view/ktb-stage-overview/ktb-stage-overview.component.html

```diff
@@ -1,29 +1,30 @@
-<div class="ktb-stage-overview container" fxFlexFill *ngIf="project as project">
+<div class="ktb-stage-overview container" fxFlexFill *ngIf="project$ | async as project">
   <div>
     <p><span [textContent]="project.stages.length"></span> Stages</p>
     <dt-filter-field
       class="mb-3"
       [dataSource]="_dataSource"
       [filters]="filter"
-      (filterChanges)="filterChanged($event)"
+      (filterChanges)="filterChanged(project, $event)"
       label="Filter by service"
       aria-label="Filter by service"
       clearAllLabel="Clear all"
     ></dt-filter-field>
-    <div *ngIf="!isTriggerSequenceOpen">
-      <button dt-button class="mb-3" (click)="isTriggerSequenceOpen = true" uitestid="keptn-trigger-button-open">
-        <dt-icon name="flash"></dt-icon>Trigger a new sequence
-      </button>
-    </div>
-    <div class="mb-3 pr-1" *ngIf="isTriggerSequenceOpen">
+    <div *ngIf="isTriggerSequenceOpen; else triggerSequenceClosed">
       <ktb-trigger-sequence
         [projectName]="project.projectName"
-        [stage]="selectedStage?.stageName"
+        [stage]="selectedStageName$ | async"
```

Comment:
You know that this observable is subscribed two times in the view right? But it might not be a problem when looking at the pipe (because it is executed twice then).